src/fparser/two/Fortran2003.py

```diff
@@ -11032,3 +11033,5 @@ class Scalar_%s(Base):
 ''' % (n, n))
 
 # EOF
+
```

Comment:
I think you can remove the '# EOF' above.

Reply:
Done